Functions/BlobStorageOptions.cs

```diff
@@ -1,7 +1,8 @@
 ﻿namespace Functions
 {
-    public class BlobStorageOptions
+    public class BlobStorageOptions : IOptions<BlobStorageOptions>
     {
         public string BlobContainerName { get; set; }
+        public BlobStorageOptions => this;
```

Comment:
Whoops! Was supposed to be:
```suggestion
        public BlobStorageOptions Value => this;
```